[2025-08-27 06:08:43] Starting Dual Adapter GRPO Training
Loading checkpoint shards: 100%|██████████| 4/4 [00:03<00:00,  1.13it/s]
[2025-08-27 06:08:49] Loading Data
Traceback (most recent call last):
  File "/media/nlplab/hdd1/hclt_ss/grpo/main.py", line 119, in <module>
    main()
  File "/media/nlplab/hdd1/hclt_ss/grpo/main.py", line 58, in main
    train_data = load_data(args.train_data)
  File "/media/nlplab/hdd1/hclt_ss/grpo/data.py", line 10, in load_data
    with open(dataset, 'r', encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: './dataset/함의분석_train.json'
